### DISCUSSION TITLE
Dependabot unsupported timezone

### DISCUSSION THREAD

**Discussion by yukal** [community/community#38927] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

It seems that the `Europe/Kyiv` timezone isn't supported.

I carefully read the [Configuring](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/configuring-notifications-for-dependabot-alerts) notifications for Dependabot alerts and tried to fill in my [timezone](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduletimezone), but I got an error: "**is not a valid timezone"**.

My [dependabot.yml](https://github.com/yukal/experimentals/blob/b395ce48f2cc3a8dc83c2bbeb40da403a051a5c8/.github/dependabot.yml#L11), and the [log data](https://github.com/yukal/experimentals/runs/9449227640)

**Comment by jeffwidman** [comment]
Can you file this as a bug against https://github.com/dependabot/dependabot-core/? We'll double-check the timezone library we use to see if it supports that time zone.

Currently we (the Dependabot team) aren't watching community discussions as closely as the issue tracker. That may change in the future, but we're currently at capacity on the number of inbound channels that we're monitoring.

**Reply by yukal** [reply]
@jeffwidman Just added an issue bug [here](https://github.com/dependabot/dependabot-core/issues/6132). Thank you for the help!
https://github.com/dependabot/dependabot-core/issues/6132

**Reply by jeffwidman** [reply]
To close the loop here, I fixed this internally. @yukal do you mind marking this as the accepted answer / resolved?

**Reply by yukal** [reply]
@jeffwidman Sure, it works well and we can close the loop! Is anything depends on me to close the loop? It seems that it closed already? Sorry I haven't been here for a long time
